_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,287 @@
+---
+id: a25f45
+name: Headers attribute of td element refers to cells in the same table element
+rule_type: atomic
+description: |
+  This rule checks that the headers attribute of `td` elements refer to cells in the same `table` element that have a semantic role of `columnheader` or `rowheader`.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to any [`headers`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attr-headers) attribute specified on a [`cell`](https://www.w3.org/TR/html52/tabular-data.html#cell) in a [table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) element that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Expectation
+
+The `headers` attribute of each target element is [a set of space separated IDs](https://www.w3.org/TR/html50/infrastructure.html#set-of-space-separated-tokens), each of which is an ID of an element in the same [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees) as the applicable elment, that:
```

Comment:
What happen if there are several element with that id?
```html
<table>
  <thead>	
    <tr>
      <th id="header1">Projects</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td headers="header1">15%</td>
    </tr>
<tr>
      <td id="header1">25%</td>
</tr>
  </tbody>
</table>
```

Reply:
This could be more developer error than an actual scenario. I did VoiceOver testing and in some cases one is announced or none :/. I am unsure if this is within the scope of this rule.

Reply:
I agree that this would be more of a global HTML error than a problem with this rule.
Maybe it's worth an assumption "This rule assume that `id` are unique within the current document" to ensure that we can write "the ID of **an** element" without any ambiguity.